model/src/main/java/io/spine/examples/shareaware/wallet/MoneyReservationSignal.java

```diff
@@ -71,4 +75,21 @@ default String withdrawalIdValue() {
     default boolean isPartOfWithdrawal() {
         return getOperation().hasWithdrawal();
     }
+
+    /**
+     * Fetches the {@code PurchaseId} from {@code OperationId} without checking for its existence.
+     *
+     * <p>It will return the default instance of {@code PurchaseId}
+     * if it is absent in {@code OperationId}.
+     */
+    default PurchaseId purchaseProcess() {
+        return getOperation().getPurchase();
+    }
+
+    /**
+     * Verifies that signal is a part of the purchase process or not.
```

Comment:
"Tells whether this signal is a part of the purchase process, or not".

"Verification" is something different.